src/client/editor/Editor.js

```diff
@@ -378,6 +378,26 @@ export default class Editor {
     return exportScene(this.scene);
   }
 
+  async extendScene(inheritedURI) {
+    const extendSceneDef = {
+      entities: {},
+      inherits: inheritedURI
+    };
+
+    this.deselect();
+    this._resetHelpers();
+    const ancestors = [];
+    const scene = await loadSerializedScene(extendSceneDef, inheritedURI, this.addComponent, true, ancestors);
+
+    scene.userData._ancestors = ancestors;
+
+    this._setSceneInfo(scene, null);
+    this.scenes = [this.sceneInfo];
+    this._setSceneDefaults(scene);
+    this._setScene(scene);
+    this._addDependency(null, scene);
```

Comment:
It doesn't make sense to add a dependency on a null uri, it should be inheritedURI instead of null.